doc/operator-guide/deployment.rst

```diff
@@ -2,5 +2,43 @@
 Deployment
 ###########
 
-* Database
-* REST API Server
+Database
+========
+
+- Deployments of the Consolidated Database are currently located at
+
+-  Summit
+-  USDF (+ dev, use the same underlying database, a replication of Summit)
+-  Base Test Stand (BTS)
+-  Tucson Test Stand (TTS)
+
+- Updates to these deployments may be needed when there are edits to the schema for any of the cdb_* tables defined in <link to> sdm_schemas.
+
+Tools:
+------
+
+- Argo-CD
+- LOVE
+- Felis
+
+Repositories:
+-------------
+
+- ``phalanx`` (https://github.com/lsst-sqre/phalanx)
+- ``sdm_schemas`` (https://github.com/lsst/sdm_schemas)
+- ``consdb`` (https://github.com/lsst-dm/consdb)
+
+Access needed:
+--------------
+
+- NOIRLab VPN
+- Summit VPN
+- USDF
+
+Process:
+--------
+
```

Comment:
Shouldn't this section have content if the PR is for adding deployment process documentation?